web/locales/en/plugin__network-observability-plugin.json

```diff
@@ -18,6 +19,7 @@
   "Specify a protocol following one of these rules:": "Specify a protocol following one of these rules:",
   "A protocol number like 6, 17": "A protocol number like 6, 17",
   "A IANA name like TCP, UDP": "A IANA name like TCP, UDP",
+  "Specify kubernetes label containing any alphanumeric, minus or dot caracter": "Specify kubernetes label containing any alphanumeric, minus or dot caracter",
```

Comment:
the usual term for '-' is hyphen.
For instance, here's a very similar message I found in console:
`Name must consist of lower-case letters, numbers and hyphens. It must start with a letter and end with a letter or number.`

Reply:
also there's a 'h' missing in "character" :)